### DISCUSSION TITLE
Field's index

### DISCUSSION THREAD

**Discussion by manson** [pocketbase/pocketbase#84] [Q&A] [answered]
Cannot find any fields indexing possibility. Did I miss this?

**Comment by ganigeorgiev** [comment, marked as answer]
Currently you cannot define indexes from the admin UI. Single field/column indexes are in the roadmap.

In the meantime, you can define indexes (if you really need them) via DB migration (or if you are using the prebuilt base binary - with the sqlite3 cli).
